languages/elixir/exercises/concept/anonymous-functions/.docs/instructions.md

```diff
@@ -0,0 +1,67 @@
+In this exercuse, you'll be creating some auxillary functions in order to simplify to creation of closure functions.
```

Comment:
Let me know if you disagree, but to me this is not really a story/theme. For that, I feel that it is too low-level. Could you perhaps consider working on a story/theme that lends it to using anonymous functions somehow?

Reply:
I can work on it, but I'm not sure there is much compelling to indroduce it.  From a cursory search of lambda functions an closures, this is about what I saw on most tutorials.

The conflict I am seeing here is: anonymous functions and closures are necessary for higher-order functions to manipulate lists/maps.  Teaching anonymous functions and closures in a limited way without drawing on much else really limits the story ideas that can go along with this.

Reply:
I have added a theme, take a peek when the revision is pushed?